src/components/standardUi/Table.vue

```diff
@@ -4,81 +4,94 @@
   <table>
     <thead>
       <tr>
-        <th v-for="(value, name, index) in items[0]" :key="value" @click="sortBy(index)" :class="{ active: sortKey === index }">
-          {{name}} <span class="arrow" :class="sortOrders[index] > 0 ? 'asc' : 'dsc'"/>
+        <th
+          v-for="(value, name, index) in items[0]"
+          :key="value"
+          @click="sortBy(index)"
+          :class="{ active: sortKey === index }"
+        >
+          {{ name }}
+          <span class="arrow" :class="sortOrders[index] > 0 ? 'asc' : 'dsc'" />
         </th>
       </tr>
     </thead>
     <tbody>
-      <tr v-for="entry in filteredItems" :key="entry">
-        <td v-for="key in entry" :key="key"> {{key}} </td>
+      <tr
+        v-for="entry in filteredItems"
+        :key="entry"
+      >
+        <td v-for="cell in entry" :key="cell" @click="$emit('cell-clicked', cell)">{{ cell.displayName }}</td>
       </tr>
     </tbody>
   </table>
 </template>
 
 <script lang="ts">
+import { LocationType } from "@/classes/LocationType";
+import { TableCell } from "@/classes/TableCell";
```

Comment:
maybe even kijken of deze class een LocationTable moet worden of dat het nog generiek te maken is.